build.rs

```diff
@@ -14,28 +14,21 @@ fn main() {
     let minor = (gdal_version - major * 1000000) / 10000;
     let patch = (gdal_version - major * 1000000 - minor * 10000) / 100;
```

Comment:
is that line calculating patch still needed ?

Reply:
It's still printed in the panic message below, just in case. Probably doesn't hurt to have it.

Reply:
ah ok, nevermind, I missed that use